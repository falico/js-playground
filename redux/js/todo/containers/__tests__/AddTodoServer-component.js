import React from 'react'
import renderer from 'react-test-renderer'
import AddTodoServer from '../AddTodoServer'
import Store from '../../__mocks__/store'

describe('AddTodoServer', () => {
  it('renders', () => {
    const component = renderer.create(
      <AddTodoServer store={Store({})} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
