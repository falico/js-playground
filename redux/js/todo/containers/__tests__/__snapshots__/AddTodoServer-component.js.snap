exports[`AddTodoServer renders 1`] = `
<div>
  <form
    onSubmit={[Function]}>
    <input />
    <button
      type="submit">
      Add Todo (Server)
    </button>
  </form>
</div>
`;
